security-policies/bundle/compliance/cis_azure/rules/cis_2_1_20/rule.rego

```diff
@@ -0,0 +1,28 @@
+package compliance.cis_azure.rules.cis_2_1_20
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import future.keywords.if
+import future.keywords.in
+
+finding = result if {
+	# filter
+	data_adapter.is_security_contacts
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(notification_alert_high),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default notification_alert_high = false
+
+notification_alert_high if {
+	# Ensure at least one Security Contact Settings exists and alertNotifications severity is set to high.
+	some security_contact in data_adapter.resource
+
+	security_contact.name == "default"
+	lower(security_contact.properties.alertNotifications.state) == "on"
+	lower(security_contact.properties.alertNotifications.minimalSeverity) == "high"
```

Comment:
> Ensure that the `Notify about alerts with the following severity (or higher):` setting is checked and set to `High`

What does the "or higher" refer to in that context?

Reply:
<img width="962" alt="Screenshot 2024-01-31 at 7 17 33 PM" src="https://github.com/elastic/cloudbeat/assets/1380039/483cc871-4737-4a7c-8c51-4235e2ec6e97">
(unless I misunderstood the question) This refers to the label text in the UI.